package.json

```diff
@@ -185,22 +185,21 @@
     "script": "babel-node --extensions .js,.ts $1",
     "sequelize": "babel-node --extensions .js,.ts -- ./node_modules/.bin/sequelize --config config/sequelize-cli.js",
     "start": "node ./dist/index.js",
-    "test": "TZ=UTC nyc --reporter=lcov mocha",
+    "test": "TZ=UTC nyc mocha",
     "test:clean": "rm -rf .nyc_output coverage",
     "test:coverage": "codecov",
-    "test:graphql": "TZ=UTC mocha test/server/graphql",
-    "test:lib": "TZ=UTC mocha test/server/lib",
-    "test:models": "TZ=UTC mocha test/server/models",
-    "test:payment-providers": "TZ=UTC mocha test/server/paymentProviders",
-    "test:routes": "TZ=UTC mocha test/server/routes",
-    "test:scripts": "TZ=UTC mocha test/scripts",
-    "test:watch": "TZ=UTC nyc mocha --watch",
```

Comment:
I missed that this was removed, I use this command very often to do TDD. @znarf are you ok if I add it back?

Reply:
Actually it's fine, I always specify a file when doing that so adding `--watch` at the end is not more complex

Reply:
Sorry, let's bring it back!